modules/ecs/service.json.tftpl

```diff
@@ -0,0 +1,33 @@
+[
+  {
+    "name": "${namespace}",
+    "image": "${aws_ecr_repository}:${aws_ecr_tag}",
+    "essential": true,
+    "memory": ${web_container_memory},
+    "cpu": ${web_container_cpu},
```

Comment:
If there is just a single container per ECS task, it is not necessary to define the container's CPU & memory as these 2 attributes are optional
https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#container_definition_memory
> If you're using the Fargate launch type, this parameter is optional.

in that case, defining the CPU & memory for the ECS task is enough
https://github.com/sanG-github/nimble-devops-ic-infrastructure/pull/30/files#diff-e7a6edcea4732d8fc16fe257b951dbb5fd6a4ca57cf317fc3147d1ef21e70b43R49-R50

Let's leave the task to allocate the CPU & memory for the container

Reply:
Thanks for sharing it, I really appreciate it and now understand about how ECS Task/container works.

It was removed in cfa4c84 ✅